backend/covexit/urls.py

```diff
@@ -14,11 +14,12 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from rest_framework.routers import DefaultRouter
 from django.urls import path, include
-
+from oscarapi.urls import urlpatterns as oscarapi_urls
+from covexit.account.api.urls import urlpatterns as account_patterns
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('api/v1/', include('oscarapi.urls')),
+    path('api/v1/', include(oscarapi_urls)),
+    path('api/v1/', include(account_patterns)),
```

Comment:
Since you redefined the api root, should we define our own API endpoints within there? So they all are shown on the http overview?

Reply:
Yes, new endpoints that aren't already in OscarAPI are going to have to be added to the root view to show up on the overview.